src/main/java/com/example/TicketApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.example;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.time.LocalDateTime;
+
+
+@SpringBootApplication(scanBasePackages = "com.example")
```

Comment:
when you class with `@SpringBootApplication` is placed in root folder, and all other classes are placed at more deep level, you do not need to specify `scanBasePackages ` additionally. Spring will scan this root package and all subpackages

Reply:
+